src/OSPSuite.Core/CoreRegister.cs

```diff
@@ -115,14 +114,8 @@ private static void registerConverters(IContainer container)
             scan.IncludeNamespaceContainingType<IObjectConverter>();
             scan.WithConvention<RegisterTypeConvention<IObjectConverter>>();
             scan.ExcludeType<ObjectConverterFinder>();
-            scan.ExcludeType<DimensionMapper>();
-            scan.ExcludeType<FormulaMapper>();
-            scan.ExcludeType<SerializationDimensionFactory>();
          });
          container.Register<IObjectConverterFinder, ObjectConverterFinder>(LifeStyle.Singleton);
-         container.Register<IDimensionMapper, DimensionMapper>(LifeStyle.Singleton);
-         container.Register<IFormulaMapper, FormulaMapper>(LifeStyle.Singleton);
-         container.Register<ISerializationDimensionFactory, SerializationDimensionFactory>(LifeStyle.Singleton);
```

Comment:
All those objects are somehow still used in MoBI and create tension. I'd like to simplify the code a bit and it starts with removing code that is older then us :)